### PR TITLE
#160843647 Fix assertion issues in the tests.

### PR DESCRIPTION
**Pull request by gitaumoses4**
## What does this PR do?
Fixes the assertion issues in the develop branch for the tests.

## Description of task to be completed
Have the tests passing.

## How can this be manually tested?
Clone the repository to your computer
```
git clone https://github.com/andela/ah-code-blooded.git
```
Cd into the directory
`cd ah-code-blooded`
Create a virtual environment and activate it
`python3 -m venv venv`
`source venv/bin/activate`
Create a .env file from the `.env_example` file.

### Create the static files
`python manage.py collectstatic`

Run the tests
`python manage.py test`

## Screenshots
screen shot 2018-09-30 at 10 18 09
screen shot 2018-10-03 at 16 22 52

## Relevant Pivotal Tracker stories
[#160577595](https://www.pivotaltracker.com/story/show/160577595)

### CODE REVIEW

**Comment by esirK** [comment]
Looks good. Will be merged after https://github.com/andela/ah-code-blooded/pull/11 have been merged just to be sure everything is working.

**Comment by esirK** [comment]
@gitaumoses4 kindly fix all the failing tests now.

**Comment by gitaumoses4** [comment]
Hello @esirK 
I have fixed all the issues. The tests are now passing.

**Comment by esirK** [comment]
Have you rebased on develop??

**Comment by gitaumoses4** [comment]
Yes. That is usually the first thing I do once a PR is merged to develop. 😊

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,9 @@ install:
   - pip3 install coveralls
 
 before_script:
+  - python manage.py collectstatic
```
Comment (esirK) [inline .travis.yml:15]: Also, add a `python manage.py migrate` for the migrations.

Comment (esirK) [reply .travis.yml:15]: @gitaumoses4 kindly add the command above.

Comment (gitaumoses4) [reply .travis.yml:15]: Hello @esirK I have added the command, though I had to include the `psql -c 'create database'` command in order for it to function.

**Pull request merged**